src/main/kotlin/app/revanced/patches/youtube/player/overlaybuttons/OverlayButtonsPatch.kt

```diff
@@ -67,6 +68,15 @@ object OverlayButtonsPatch : BaseResourcePatch(
         required = true
     )
 
+    // Option to change top buttons
+    private val ChangeTopButtons by booleanPatchOption(
+        key = "ChangeTopButtons",
+        default = false,
+        title = "Change top buttons",
+        description = "Change the icons at the top of the player.",
```

Comment:
@KobeW50 I'd like to request a review to ensure the description of the new patch options is adequate

Reply:
I will merge the PR first
If changes are needed later, please open a new PR